addons/strongerFlashlights/CfgWeapons.hpp

```diff
@@ -0,0 +1,306 @@
+class cfgWeapons
+{
+    class InventoryFlashLightItem_Base_F;    
+    
+    class cup_acc_flashlight; //CUP Flashlight -surefire
+    
+    class ItemCore;
+    class ACE_acc_pointer_green;
+    class acc_flashlight_pistol;
+    
+/////////////////////////////////////////
+//////////Multimode Flashlight///////////
+/////////////////////////////////////////    
+    
+    class GVAR(CUP_LED_flashlight_Beam): cup_acc_flashlight
```

Comment:
Brace has to be on same line

Same for all subsequent braces, if we have `class BLAH` we want `class BLAH {`